docs/diagrams/ModelClassDiagram.puml

```diff
@@ -19,10 +19,12 @@ Class Name
 Class Phone
 Class Tag
 Class PostalCode
+'@@author lokidoki102
```

Comment:
I think should be //@@author {yourGithubUsername} ?

Reply:
The guide mentioned something about using the comment syntax according to the file type. So I used ' instead of //.
![image](https://github.com/AY2324S2-CS2103-F09-1/tp/assets/42115432/74b8c45c-1095-426a-aad2-15314279f0a2)

Reply:
Alright I understand now